volumes/frus1981-88v11.xml

```diff
@@ -6327,9 +6339,9 @@
                             number of strategic systems, which makes it all the more one-sided in
                             the Soviets’ favor.</p>
                         <closer>
-                            <signed><persName corresp="#p_LPBremer3_1">L. Paul Bremer,
-                                    III</persName>
-                                <affiliation>Executive Secretary</affiliation></signed>
+                            <signed><persName corresp="#p_LPBremer3_1"><hi rend="strong">L. Paul
+                                        Bremer, III</hi></persName><lb/><hi rend="italic">Executive
+                                    Secretary</hi></signed>
```

Comment:
@vak2ve A space before and after an lb elements is needed, unless the lb element breaks a word.